students/17th/team3/sungjounjang/user/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+class Accounts(self, models.Model):
+    email        = models.EmailField(max_length=100)
```

Comment:
```suggestion
    email        = models.EmailField(max_length=100)
```
 오 EmailField를 찾아서 쓰셨군요! 잘하셨습니다👍

Reply:
ㅋㅋㅋ감사합니다!😂